decidim-initiatives/app/mailers/decidim/initiatives/initiatives_mailer.rb

```diff
@@ -51,27 +51,6 @@ def notify_state_change(initiative, user)
         end
       end
 
-      # Notify an initiative requesting technical validation
-      def notify_validating_request(initiative, user)
```

Comment:
If this method is deleted, we have to take care about:
- decidim-initiatives/spec/mailers/previews/initiatives_mailer_preview.rb
where this method is being used:
![image](https://user-images.githubusercontent.com/9702463/81057284-6bb94d00-8ecc-11ea-9f84-8aa2df6614b2.png)
although it seems that 'notify_state_change_to_validating' is neither used anywhere.

Reply:
Good catch!
It is a preview, this is why there's no reference in the code https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails

I've removed it as the email no longer exists!